### PR TITLE
chore(external docs): Add `loki` sink metadata

### PR DESCRIPTION
**Pull request by binarylogic**
Signed-off-by: binarylogic <bjohnson@binarylogic.com>

### CODE REVIEW

**Review by jamtur01 — APPROVED** [review]
LGTM

**Comment by jamtur01** [comment]
bors merge

File: docs/reference/components/sinks/loki.cue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	configuration: {
```
Comment (juchiast) [inline docs/reference/components/sinks/loki.cue:69]: `endpoint` is missing
https://github.com/timberio/vector/blob/dbbed8db8c7b1fd6823180e1e37168bbc99148b1/src/sinks/loki.rs#L36

**Pull request merged**